Tests/FluentAssertions.Specs/Primitives/ObjectAssertionSpecs.cs

```diff
@@ -99,6 +100,92 @@ public void When_comparing_a_numeric_and_an_enum_for_equality_it_should_throw()
             // Assert
             act.Should().Throw<XunitException>();
         }
+
+        [Fact]
+        public void A_value_is_equal_to_another_according_to_a_comparer()
```

Comment:
🤔 I took me a couple of seconds to detect the difference between this test and the next one. Maybe use a term like _untyped_ or _object_?